util/packages.go

```diff
@@ -74,8 +74,13 @@ func getPackagePaths(allPaths []string) ([]string, error) {
 			if info.IsDir() {
 				log.Printf("%-20s\t%10s\t%s", dirs[0], dirs[1], path)
 				foundPaths = append(foundPaths, path)
+				return filepath.SkipDir
 			}
-			return filepath.SkipDir // don't need to go deeper
+			// Not expected file, return nil in order to continue processing sibling directories
+			// Fixes an annoying problem when the .DS_Store file is left behind and the package
+			// is not loading without any error information
+			log.Printf("error: unexpected file: %s", path)
```

Comment:
Nit: consider changing this to a `warning` instead of an `error` and also mention that we're simply ignoring the unexpected file. That way the person looking at these logs knows this is not a severe problem to worry about.